test/functional/system/blocks/verify/verify.js

```diff
@@ -148,10 +150,18 @@ function createBlock(
 		previousBlock: blocksModule.lastBlock.get(),
 		transactions,
 	});
+
 	// newBlock.id = blockLogic.getId(newBlock);
 	return newBlock;
 }
 
+const convertToBignum = transactions => {
```

Comment:
I already saw that function somewhere, can we reuse? Or if not - please move that to some common test file, as for sure we will use that more in the future.